### PR TITLE
LL-7607 Add inline app installs analytics events

### PR DESCRIPTION
**Pull request by OlivierFreyssinet**
Tracking events whenever there is an inline app install (i.e: an action that requires an app to be installed on the device so it automatically gets installed).

```
track(
   "In-line app install",
   { appName, flow },
];
```

The value of `flow` will default to "unknown" and for the main flows that I could identify and test, it will take a specific value:

#### Flows tested and working:

- [x] Add account -> inline app install -> event with prop **`flow: "add account"`**
- [x] Buy -> has no account -> add account -> inline app install -> event with prop **`flow: "buy"`**
- [x] Buy -> has account -> inline app install -> event with prop **`flow: "buy"`**
- [x] Receive -> has account -> inline app install -> event with prop **`flow: "receive"`**
- [x] Send -> has account -> inline app install -> event with prop **`flow: "send"`**
- [x] Swap -> has no account -> add account -> inline app install -> event with prop **`flow: "swap"`**

#### Flows not tested (not enough funds to test):
- [ ] Swap -> has account -> inline app install -> should send an event with prop **`flow: "swap"`**
### Type

Analytics

### Context

[LL-7607]

### Parts of the app affected / Test plan

1. Set env variable `ANALYTICS_CONSOLE=1`
2. Rebuild the app
3. Test all the aforementioned flows

[LL-7607]: https://ledgerhq.atlassian.net/browse/LL-7607?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by LFBarreto — DISMISSED** [review]

File: src/components/RootNavigator/ExchangeBuyFlowNavigator.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // @flow
-import React, { useMemo } from "react";
+import React, { useEffect, useMemo } from "react";
```
Comment (gre) [inline src/components/RootNavigator/ExchangeBuyFlowNavigator.js:2]: lint

**Review by gre — CHANGES_REQUESTED** [review]

**Review by gre — APPROVED** [review]

**Review by gre — DISMISSED** [review]
still failing. check CI https://github.com/LedgerHQ/ledger-live-mobile/runs/4102984493?check_suite_focus=true

**Comment by ofreyssinet-ledger** [comment]
@gre could you redo a quick review please now that CI is passing? 🙏

**Comment by nabil-brn** [comment]
@ofreyssinet-ledger

New swap form has been merged in develop, I resolved the conflict but you've to add analytics again on  [src/screens/Swap/FormSelection/AccountSelect.js](https://github.com/LedgerHQ/ledger-live-mobile/blob/develop/src/screens/Swap/FormSelection/AccountSelect.js).

**Comment by ofreyssinet-ledger** [comment]
@nabil-brn it should be fine now

**Review by nabil-brn — APPROVED** [review]
✅  QA-ed
---
b673116cd (HEAD -> LL-7607, upstream/LL-7607) Add analyticsPropertyFlow: "swap" in new swap when navigating to screen with potential in-app install

**Review by LFBarreto — APPROVED** [review]

**Pull request merged**